connectors/jira/src/main/java/com/vmware/connectors/jira/JiraController.java

```diff
@@ -230,7 +233,7 @@ private void addCommentsField(CardBody.Builder cardBodyBuilder, List<String> all
                     .setType(CardBodyFieldType.COMMENT);
 
             cardFieldBuilder.addContent(ImmutableMap.of("text", allComments.get(0)));
-            if (allComments.size() > 1) {
+            if (allComments.size() > COMMENTS_SIZE) {
```

Comment:
I've had a better idea. Change the value of `COMMENTS_SIZE` to 2 and replace the code with this:
```java
allComments.stream()
        .limit(COMMENTS_SIZE)
        .forEach(comment -> cardFieldBuilder.addContent(ImmutableMap.of("text",comment)));
```

Reply:
It is a good idea. Changed the code to above format...